EIPS/eip-3155.md

```diff
@@ -0,0 +1,241 @@
+---
+eip: 3155
+title: EVM trace specification
+author: Martin Holst Swende (@holiman), Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: https://ethereum-magicians.org/t/eip-3155-create-evm-trace-specification/5007
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): Interface
+created: 2020-12-07
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+This EIP introduces a new JSON standard for EVM traces. There has been a de-facto standard which is used by go-ethereum, OpenEthereum, Besu and Nethermind. This document shall document this standard as well as provide stricter guidelines to make new implementations easier.
```

Comment:
Not a de-facto standard, other tracing formats are supported by OE, Besu and Nethermind that are friendlier to smart contract developers.  It is a common format.